en/docs/develop/service-provider-configurations-used-with-apis.md

```diff
@@ -18,6 +18,11 @@ This section guides you through the configurations you can include in a service
         <td>
             <ul>
                 <li><b>issuer</b> <code>[String]</code> <br>This specifies the issuer. This is the <code>             &lt;saml:Issuer&gt;            </code> element that contains the unique identifier of the service provider. This is also the issuer value specified in the SAML Authentication Request issued by the service provider. When configuring single-sign-on across Carbon servers, ensure that this value is equal to the <code>ServiceProviderID</code> value mentioned in the <code>             &lt;IS_HOME&gt;/repository/conf/deployment.toml</code> file of the relying party's WSO2 Identity Server.</li>
+                <li><b>issuerQualifier</b> <code>[String]</code> <br>
+                    <p>This value is needed only if you have to configure multiple SAML SSO inbound authentication configurations for the same Issuer value. When a Service Provider Qualifier is defined here, it will be appended to the end of the Issuer value when registering the SAML SP in the Identity Server.</p> 
+                    <p>You can configure a number of SAML SPs with the same Issuer and different Service Provider Qualifiers.</p>
+                    <p>When a Service Provider Qualifier is defined, the issuer of the SAML SSO authentication request is the value specified as the Issuer in the configuration (ex : `travelocity.com` ). The service provider qualifier value should be sent as a query parameter, spQualifier with the HTTP request in the following format.</p>
+                </li>
```

Comment:
When a service provider qualifier is defined, the issuer that is defined in the configuration, e.g., `travelocity.com` will be set as the issuer of the SAML SSO authentication request. The `service provider qualifier` should be sent as the query parameter `spQualifier` along with the HTTP request in the following format:

Reply:
Fixed